### ISSUE TITLE
No WYSIWYG editor shows up when using wide-theme

### ISSUE THREAD

**Issue by mayinx** [melvinsembrano/cms-fortress#82]
Hi there! (awesome gem by the way) I just had a couple of minor issues when using the wide-theme: The WYSIWYG editor buttons only appear, when I change the theme in the initializer to `:default` (explicitly - commenting `config.theme = :wide` wasn't enough)... Same goes for the ability to switch Layouts when editing / creating pages, which only works using the default-theme (i.e. layouts can be changed in the Layout-Dropdown using the wide-theme - but the corresponding form fields don't update accordingly - seems to be a javascript issue).

In case it helps / matters: I installed cms-fortress v1.3.9 on a rails 4.2.1 / ruby 2.1.5 project with an existing Comfortable Mexican Sofa (v1.12.8) installation (using `rails generate cms:fortress --no-comfy` )

**Comment by nodu** [comment]
+1  
What's the deal with this?  The WYSIWYG/Rich text editor does not show when on the wide theme using {{cms:page:label:rich_text}}  

@mayinx Did you find a solution?

**Comment by andywenk** [comment]
@nodu You should use

    {{ cms:page:content:rich_text }}

Does it fix the not showing editior?

**Comment by nodu** [comment]
@andywenk No it doesn't fix the issue.  I'm noticing a 
`Uncaught ReferenceError: tinymce is not defined` when using the wide.

**Comment by andywenk** [comment]
@nodu ah ok. tinymce (the JavaScript for the Editor) is missing. I saw that you added the option `--no-comfy` so you have an already installed Comfy?

**Comment by nodu** [comment]
Yes, I installed Comfy first, then found Fortress to solve my Auth
concerns, etc.

How do I include the JS?  I've put `//= require tinymce` into my
Application.js

**Comment by andywenk** [comment]
@nodu I updated my comment ...

**Comment by nodu** [comment]
@andywenk For testing, I created a new application not installing Comfy and directly install CMS Fortress.  First login complains about 
`Uncaught ReferenceError: tinymce is not defined`

How do I proceed?

**Comment by nodu** [comment]
If a vanilla install of CMS fortress does not work with the wide theme, CMS has a huge issue.

**Comment by ferrisoxide** [comment]
@nodu this may help, if you don't care which WYSIWYG editor you end up using. 

In your app, modify the `/config/cms/fortress/global_settings.yml` file to read something like this:

```
default: &default
  title: "CMS Fortress"
  title_link: "#"
  use_tinymce: false
```

You may also need to run `[bundle exec] rake assets:clobber` after making the change (and before you start the app up again).

This should pull in the Redactor WYSIWYG editor, which seems to be the default in Comfy now (some mention of an OEM license in the link @andywenk provided (https://github.com/comfy/comfortable-mexican-sofa/releases/tag/v1.12.5).

Hope that works for you. I have the Redactor editor working here fine on a new CMS-Fortress, but I haven't been able to test any of the image library support (can't currently install ImageMagick - unrelated `brew` issue).

**Comment by corthmann** [comment]
Great tip @ferrisoxide ! Disabling `tinymce` worked for me as well. CMS-Fortress was unable to add images in the richtext editor before `tinymce` was disabled.

**Comment by nodu** [comment]
Legend! Thanks @ferrisoxide, worked like a charm. This should be added to the Readme/Wiki

**Comment by andywenk** [comment]
@ferrisoxide yes the licences is granted for Redactor as long as you use it in Comfy only. because cms-fortress is a plugin for Comfy you are fine.

To all others. You are right. Since Comfy changed to Redactor as the default editor, cms-fortress may have an issue with enabeling TinyMCE it by default. My suggestion is to remove TinyMCE completely and use whatever Comfy provides. Less differentiation between Comfy and a plugin is always better. 

I have to admit, that I did not set up a vanilla cms-fortress project for a longer time. I hope I can dig into it and provide some patches. The problem ist that we heavily customized both Comfy and cms-fortress so that there are huge differences and it will take some time to merge it - if at all mergeable and wanted.

@melvinsembrano what is your opinion on removing TinyMCE.

Again to all others - patches are welcome :dancers:

**Comment by javiros** [comment]
@ferrisoxide thanks a million, after hours of fiddling with it your post sorted me!

**Comment by andywenk** [comment]
I have installed a vanilla Comfy with cms-fortress and ran into the same problem you folks mentioned above. I will file a bug and try to resolve it in a PR asap ...

**Comment by melvinsembrano** [comment]
This is now fixed on version 1.3.12. Redactor is now the default WYSIWYG editor.  Support of TinyMCE is temporarily disabled until the feature is fully fixed and tested.
